spec/features/openid_connect/phishing_resistant_required_spec.rb

```diff
@@ -19,27 +19,59 @@
 
     context 'user has aal3 auth configured' do
       it 'sends user to authenticate with AAL3 auth' do
-        sign_in_before_2fa(user_with_aal3_2fa)
+        sign_in_before_2fa(user_with_phishing_resistant_2fa)
 
         visit_idp_from_ial1_oidc_sp_requesting_aal3(prompt: 'select_account')
         visit login_two_factor_path(otp_delivery_preference: 'sms')
         expect(current_url).to eq(login_two_factor_webauthn_url)
       end
 
       it 'does not allow an already signed in user to bypass AAL3 auth' do
-        sign_in_and_2fa_user(user_with_aal3_2fa)
+        sign_in_and_2fa_user(user_with_phishing_resistant_2fa)
         visit_idp_from_ial1_oidc_sp_requesting_aal3(prompt: 'select_account')
 
         expect(current_url).to eq(login_two_factor_webauthn_url)
       end
     end
   end
 
-  describe 'ServiceProvider configured to default to AAL3 authentication' do
+  describe 'OpenID Connect requesting phishing-resistant authentication' do
     context 'user does not have aal3 auth configured' do
       it 'sends user to set up AAL3 auth' do
         user = user_with_2fa
 
+        visit_idp_from_ial1_oidc_sp_requesting_phishing_resistant(prompt: 'select_account')
+        sign_in_live_with_2fa(user)
+
+        expect(current_url).to eq(authentication_methods_setup_url)
+        expect(page).to have_content(t('two_factor_authentication.two_factor_aal3_choice'))
```

Comment:
**nitpick:** looks like we'll also have some I18n keys to rename as well 😭

Reply:
yeah, planning to tackle that in a future PR to keep the diff smaller here since it's already kind of unwieldy